cli/flox-rust-sdk/src/providers/buildenv.rs

```diff
@@ -271,6 +271,7 @@ impl BuildEnvNix {
     /// this function is currently assumes that the package is from the nixpkgs base-catalog.
     /// Currently the type is distinguished by the [LockedPackageCatalog::locked_url].
     /// If this does not indicate a nixpkgs package, the function will currently panic!
+    #[instrument(skip(self, client), fields(progress = format!("Realising package '{}'", locked.install_id)))]
```

Comment:
suggestion: I don't think we should use the word `Realising` in user facing messages. No one will know what that means

Reply:
Not only that but we decided a long time ago to standardize on American English

Reply:
I think realiße is fine, considering it is contextualized with either "building ..." or "downloading ...".

Might go with "produce" for now, alternatively "acquire"?